include/deal.II/grid/reference_cell.h

```diff
@@ -381,8 +414,53 @@ namespace ReferenceCell
             get_bit(face_orientation, 2),
             get_bit(face_orientation, 1));
         }
+
+        ReferenceCell::Type
+        face_reference_cell_type(const unsigned int face_no) const override
+        {
+          (void)face_no;
+          return ReferenceCell::Type::Quad;
+        }
       };
 
+      /**
+       * Return for a given reference-cell type @p the right Info.
+       */
+      inline const ReferenceCell::internal::Info::Base &
+      get_cell(const ReferenceCell::Type &type)
+      {
+        static ReferenceCell::internal::Info::Base   gei_invalid;
+        static ReferenceCell::internal::Info::Vertex gei_vertex;
+        static ReferenceCell::internal::Info::Line   gei_line;
+        static ReferenceCell::internal::Info::Quad   gei_quad;
+        static ReferenceCell::internal::Info::Hex    gei_hex;
+
+        switch (type)
+          {
+            case ReferenceCell::Type::Vertex:
+              return gei_vertex;
+            case ReferenceCell::Type::Line:
+              return gei_line;
+            case ReferenceCell::Type::Quad:
+              return gei_quad;
+            case ReferenceCell::Type::Hex:
+              return gei_hex;
+            default:
+              Assert(false, StandardExceptions::ExcNotImplemented());
+              return gei_invalid;
+          }
+      }
+
+      /**
+       * Return for a given reference-cell type @p and face number @p face_no the
+       * right Info of the @p face_no-th face.
+       */
+      inline const ReferenceCell::internal::Info::Base &
+      get_face(const ReferenceCell::Type &type, const unsigned int face_no)
+      {
+        return get_cell(get_cell(type).face_reference_cell_type(face_no));
+      }
```

Comment:
@drwells How do you like this way? I have introduced two utility functions.

Reply:
:+1: